plugins/modules/inventory_intent.py

```diff
@@ -2813,9 +2892,9 @@ def verify_diff_merged(self, config):
             if udf_exist:
                 self.status = "success"
                 msg = "Global UDF {0} created and verified successfully".format(field_name)
-                self.log(msg)
+                self.log(msg, "INFO")
             else:
-                self.log("Playbook paramater doesnot match with the Cisco DNA Center means creating Global UDF task not executed successfully.")
+                self.log("Playbook paramater doesnot match with the Cisco Catalyst Center means creating Global UDF task not executed successfully.", "INFO")
```

Comment:
self.log("Mismatch between playbook parameter and Cisco Catalyst Center detected, indicating that the task of creating Global UDF may not have executed successfully.", "INFO")

Reply:
added.